### PR TITLE
Change from `Why?` to `Why DiscordJS?`

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Change from `Why?` to `Why DiscordJS?` on the index page.
  - IMO, I think this way looks better as it doesn't look so empty.

### CODE REVIEW

**Comment by hickatheworld** [comment]
Such a change would make more sense if it was written `discord.js` instead of `DiscordJS`

File: src/pages/index.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@
 			<Codeblock :code="exampleCode" />
 		</div>
 		<div>
-			<h2>Why?</h2>
+			<h2>Why DiscordJS?</h2>
```
Comment (xHyroM) [inline src/pages/index.vue:46]: Why discord.js ?

**Review by xHyroM — CHANGES_REQUESTED** [review]

**Comment by iCrawl** [comment]
Meh.

**Comment by Panquesito7** [comment]
Sorry for not updating in the right time. Should I change it to `discord.js`?

**Pull request closed without merging**